hub/core/query/query.py

```diff
@@ -5,8 +5,11 @@
 import json
 import numpy as np
 
+from hub.core.index.index import Index
 from hub.core.tensor import Tensor
 
+enable_cache = True
```

Comment:
This seems to be always True.. can we remove it?

Reply:
It is outside variable to disable feature.